src/platform/silabs/wifi/SiWx/ncp/efx32_ncp_host.c

```diff
@@ -135,6 +98,45 @@ static void efx32_spi_init(void)
     GPIO_ExtIntConfig(INTERRUPT_PIN.port, INTERRUPT_PIN.pin, INTERRUPT_PIN.pin, true, false, true);
 }
 
+Ecode_t si91x_SPIDRV_MTransfer(SPIDRV_Handle_t handle, const void * txBuffer, void * rxBuffer, int count,
+                               SPIDRV_Callback_t callback)
+{
+    USART_TypeDef * usart = handle->initData.port;
+    uint8_t * tx          = (txBuffer != NULL) ? (uint8_t *) txBuffer : dummy_buffer;
+    uint8_t * rx          = (rxBuffer != NULL) ? (uint8_t *) rxBuffer : dummy_buffer;
+
+    if (count < 16)

```

Comment:
nit: can you add a comment explaining that when we have less than 16 bytes to transfer it is faster to interact with the usart buffers directly rather than using the DMA transfer

Reply:
Added comment